listenbrainz/db/testing.py

```diff
@@ -32,16 +32,10 @@ def init_db(self):
         db.run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_indexes.sql'))
 
     def drop_tables(self):
-        with db.engine.connect() as connection:
-            connection.execute('DROP TABLE IF EXISTS "user"              CASCADE')
-            connection.execute('DROP TABLE IF EXISTS listen              CASCADE')
-            connection.execute('DROP TABLE IF EXISTS listen_json         CASCADE')
-            connection.execute('DROP TABLE IF EXISTS api_compat.token    CASCADE')
-            connection.execute('DROP TABLE IF EXISTS api_compat.session  CASCADE')
```

Comment:
`listen` and `listen_json` aren't being used anymore but the PostgresListenStore tests depend upon them.

I'll update the PR with `DROP SCHEMA CASCADE` for the schemas and drop the remaining tables normally.